Gemfile.lock

```diff
@@ -502,7 +523,6 @@ DEPENDENCIES
   kaminari
   money
   neat (= 1.7.2)
-  omniauth (< 2)
```

Comment:
This pin was placed in https://github.com/artsy/convection/pull/945. Removing, because `artsy-auth` v0.2.0 addresses the problem experienced with `omniauth` v2.